tests/e2e/modules_position_spec.js

```diff
@@ -33,10 +33,9 @@ describe("Position of modules", function () {
 			position = positions[idx];
 			className = position.replace("_", ".");
 			it("show text in " + position, function () {
-				return app.client
-					.waitUntilWindowLoaded()
-					.getText("." + className)
-					.should.eventually.equal("Text in " + position);
+				app.client.$("." + className).then((result) => {
```

Comment:
You're missing the return. The test case only awaits on the promise if you return it or you do an await. Otherwise, it finishes the test case before your promise resolves or rejects.